packages/formatters-util/documentation/InputMaskExample.tsx

```diff
@@ -8,11 +8,12 @@ import { registerWithMasks } from "../src";
 import "./input-mask-example.scss";
 
 interface Skjema {
```

Comment:
Endret navnet på de som ligger inne her for at de ikke skal hete akkurat det samme som det de gjør inne i `registerWithMask.ts` (gikk selv på en liten smell der)